### ISSUE TITLE
Add a way to communicate between team members

### ISSUE THREAD

**Issue by Aadesh-Baral** [hotosm/tasking-manager#5365]
Currently, the only way for teams to communicate is for team managers to send a message to the entire team from the team edit page. Create a section similar to the project chat on the project detail page on the `teams/<team_id>/membership/` page where team members can communicate with one another.

**Comment by martien-176** [comment]
Good idea. 

I guess only team members can take part in the discussion. 

But can non-team-members read the discussion(s)?

**Comment by Aadesh-Baral** [comment]
@martien-176 I agree that only team members should be allowed to participate in discussions  In my opinion, read access can be left open for all users on public teams and limited to team members on private teams.
